### ISSUE TITLE
crlfmt wants a 101-characters line

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/crlfmt#9]
It doesn't like:
```
func (p *planner) ShowConstraints(
	ctx context.Context, n *parser.ShowConstraints,
) (planNode, error) {
```

and it wants:

```
func (p *planner) ShowConstraints(ctx context.Context, n *parser.ShowConstraints) (planNode, error) {
```

Which has 101 chars.

Can we please take this opportunity to also look at #5 and allow wrapping at less than 100 chars. I don't know how others feel, but this is eating a piece of my liver every time.

The output:
```
    	style_test.go:389: /Users/andrei/work/src/github.com/cockroachdb/cockroach/pkg/sql/show.go:719
    	style_test.go:389: -func (p *planner) ShowConstraints(
    	style_test.go:389: -	ctx context.Context, n *parser.ShowConstraints,
    	style_test.go:389: -) (planNode, error) {
    	style_test.go:389: +func (p *planner) ShowConstraints(ctx context.Context, n *parser.ShowConstraints) (planNode, error) {
    	style_test.go:389:
```

cc @jordanlewis

**Comment by danhhz** [comment]
I don't see myself working on this any time soon, so unassigning. Happy to review a PR though

**Comment by dt** [comment]
I think I fixed this in #12.
